### PR TITLE
Stefano latest

### PR DESCRIPTION
**Pull request by stefano-romano**
First attempt at modelling GDP via traditional ML methods.
I've looked at 4 models so far (plus some basic benchmarks): a plain linear model, random forest, gradient boosted trees (xgboost) and recurrent neural network (lstm). They all look at gdp + government balance & industry stock over the past n time steps, and try to predict the gdp at the next time step.
See the [jupyter notebook](https://github.com/improbable-research/sugarWorld/blob/stefano-latest/python/gdp_modelling.ipynb) for details. Overall xgboost and lstm perform best, but all models seem do to a decent job at capturing some of the dynamic.

### CODE REVIEW

**Comment by joncoello83** [comment]
Closing pr - keeping in separate branch
